client/src/components/modal-forms/form-components/ConfirmationForm.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+import { Box, Typography } from '@material-ui/core';
+
+import { Dialog } from '../../generic';
+import { FormButtons } from './';
+
+export const ConfirmationForm = ({ isOpen, onSubmit, onClose }) => {
```

Comment:
This component API looks like it would be a generic confirmation component, but there are a couple issues with it

- Name is `Form` but it returns a Dialog
- Hardcoded copy prevents reuse for other features

I'd suggest renaming this component and passing in the copy as a prop so it can be reused in the future